### DISCUSSION TITLE
Benchmarks using different CPUs, GPUs and RAM size.

### DISCUSSION THREAD

**Discussion by george-polevoy** [InteractiveComputerGraphics/SPlisHSPlasH#252] [Q&A]
Benchmarks could help optimising budget and improve performance of systems based on SPlisHSPlasH for CFD computation.
I need to know what CPU and GPU and RAM configuration can improve performance scale of computations.

Ideally it could be a build script that runs on bare metal, virtual environment or in docker, exports some report, preferably in json and MD format, just like BenchmarkDotNet does.
Those benchmarks in MD format are easily published on GitHub or in documentation without any modifications.

Users could contribute such benchmarks running in docker containers on their environments.

**Comment by janbender** [comment]
Sounds interesting. I will take a look at BenchmarkDotNet.
